var/spack/repos/builtin/packages/minigmg/package.py

```diff
@@ -24,16 +24,47 @@ class Minigmg(Package):
 
     version('master', sha256='1c2d27496a881f655f5e849d6a7a132625e535739f82575991c511cc2cf899ac')
```

Comment:
```suggestion
    version('master', sha256='1c2d27496a881f655f5e849d6a7a132625e535739f82575991c511cc2cf899ac', preferred=True)
```
Otherwise `local` will be installed by default. We probably want to default to the official source code instead of a fork.